src/Neo/Cryptography/Hasher.cs

```diff
@@ -9,24 +9,21 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
-namespace Neo.SmartContract.Native
+namespace Neo.Cryptography
 {
     /// <summary>
-    /// Represents the named curve used in ECDSA.
+    /// Represents hash function identifiers supported by ECDSA message signature and verification.
     /// </summary>
-    /// <remarks>
-    /// https://tools.ietf.org/html/rfc4492#section-5.1.1
-    /// </remarks>
-    public enum NamedCurve : byte
```

Comment:
This is is public, please dont just rename it. Mark it as obsolete and create another enum of your own please.

Reply:
sorry didn't catch this either. With the changes above I suggested, you will need this.

Reply:
> This is is public, please dont just rename it. 

Good point, fixed. Although I'm not sure whether anyone uses it since devpack has its own `NamedCurve`. I'll port it to devpack ASAP.